x-pack/plugins/ml/public/application/components/anomalies_table/influencers_cell.js

```diff
@@ -59,13 +59,14 @@ export class InfluencersCell extends Component {
               <EuiButtonIcon
                 size="s"
                 className="filter-button"
-                onClick={() =>
+                onClick={(e) => {
+                  e.target.blur(); // Remove focus from button so tooltip is hidden on click
```

Comment:
I see this code repeats in quite some places. I reckon it's worth creating a simple helper function 

```typescript
export const blurOnClick = (callback: Function) => (event: MouseEvent<HTMLButtonElement> => {
  (event.target as HTMLButtonElement).blur(); // Remove focus from button so tooltip is hidden on click
  callback();
}
``` 

so you can use it like 

```typescript
onClick(blurOnclick(() => {
  influencerFilter(
     influencer.influencerFieldName,
     influencer.influencerFieldValue,
     '+'
  )
}))
```

or 

```typescript
onClick(blurOnclick(influencerFilter.bind(null, influencer.influencerFieldName, influencer.influencerFieldValue, '+'))
```

Reply:
Good idea - done in https://github.com/elastic/kibana/pull/117493/commits/96145178e29fae0fbdd373cc15b1468557331fbc